R/target_market_share.R

```diff
@@ -0,0 +1,195 @@
+#' Add targets for production, using the market share approach
+#'
+#' This function calculates the portfolio-level production targets, as
+#' calculated using either the technology market share ratio `tmsr` or sector
+#' market share percentage `smsp`.
```

Comment:
We no longer output columns `tmsr` and `smsp`, right?

Reply:
Correct, updated